tests/unit/utils/get-formatted-date-range-test.js

```diff
@@ -1,23 +0,0 @@
-import { module, test } from 'qunit';
-import { getFormattedDateRange } from 'frontend-burgernabije-besluitendatabank/utils/get-formatted-date-range';
-
-module('Unit | Utility | get-formatted-date-range', function () {
```

Comment:
Why did you remove these tests?

Reply:
I heard echoes of jodi telling us testing is for the testing team